### PR TITLE
track vm errors and destroy vm by id

### PR DESCRIPTION
**Pull request by vijaykanthm28**
fix for issue #251

### CODE REVIEW

File: provision/one/cluster/machine.go [hunk 1]
```diff
@@ -101,12 +102,18 @@ func (c *Cluster) createVMInNode(opts compute.VirtualMachine, nodeAddress string
 	}
 	str := string(b)
 	spstr := strings.Split(str, ",")
-	vmres := spstr[1]
-	if err != nil {
-		return "", "", wrapErrorWithCmd(node, err, "createVM")
+
+	isSuccess, Err := strconv.ParseBool(spstr[0])
```
Comment (indykish) [inline provision/one/cluster/machine.go]: Change `Err` to `err`

File: provision/one/cluster/machine.go [hunk 2]
```diff
@@ -101,12 +102,18 @@ func (c *Cluster) createVMInNode(opts compute.VirtualMachine, nodeAddress string
 	}
 	str := string(b)
```
Comment (indykish) [inline provision/one/cluster/machine.go]: Compress those 2 lines into 1 line.

File: provision/one/cluster/machine.go [hunk 3]
```diff
@@ -124,6 +131,22 @@ func (c *Cluster) GetIpPort(opts virtualmachine.Vnc, region string) (string, str
...
 	res, err := opts.GetVm()
+	b, err := json.Marshal(res)
+
+	if err != nil {
+		return "", "", err
+	}
+	str := string(b)
+	spstr := strings.Split(str, ",")
+
+	isSuccess, Err := strconv.ParseBool(spstr[0])
```
Comment (indykish) [inline provision/one/cluster/machine.go]: Reuse `err`, compress `string(b)/split` into one line.

**Comment by vijaykanthm28** [comment]
@indykish  changes done that you have mentioned

**Comment by indykish** [comment]
Ok. cool. :+1:

**Pull request merged**